src/diffusers/loaders.py

```diff
@@ -1109,6 +1183,11 @@ def _remove_text_encoder_monkey_patch_classmethod(cls, text_encoder):
                 attn_module.v_proj = attn_module.v_proj.regular_linear_layer
                 attn_module.out_proj = attn_module.out_proj.regular_linear_layer
 
+        for _, aux_module in text_encoder_aux_modules(text_encoder):
```

Comment:
Might be better if we had a check if there are indeed auxiliary modules present.